public/ledger.js

```diff
@@ -424,6 +424,32 @@ class Ledger {
     }
   }
 
+  /**
+   * Verify a token and signature with ledger
+   *
+   * @param {Object} tokens Array of data to send to Ledger
+   *   version + uid + len(symbol) + symbol + len(name) + name + len(signature) + signature
+   *
+   * @return {Promise} Promise resolved when signature is received
```

Comment:
Only when all signatures are received, right? And it resolves with a list of the failed ones, right?